components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit-finish-ajaxprocessor.jsp

```diff
@@ -73,6 +73,8 @@
     // OIDC related properties
     boolean isRequestObjectSignatureValidated = Boolean.parseBoolean(request.getParameter("validateRequestObjectSignature"));
     boolean isIdTokenEncrypted = Boolean.parseBoolean(request.getParameter("encryptIdToken"));
+    String encryptionAlgorithm = request.getParameter("encryptionAlgorithm");
+    String encryptionMethod = request.getParameter("encryptionMethod");
```

Comment:
Can't we use already defined constants?

Reply:
These are actually the name tags defined in JSP files. I suppose we could import the constant class into the JSPs and use the constants but I don't see any actual use in that. Common practice is to use these as it is. WDYT?